config/m4/ib.m4

```diff
@@ -279,6 +288,20 @@ AS_IF([test "x$with_ib" == xyes],
                              [], [#include <infiniband/verbs_exp.h>])
            ])
 
+       # Device Memory support
+       AS_IF([test "x$with_dm" != xno],
+           [AC_TRY_COMPILE([#include <infiniband/verbs_exp.h>],
```

Comment:
why not just AC_CHECK_DECLS([ibv_exp_alloc_dm, ibv_exp_free_dm] .. ?

Reply:
because it checks for any existing declaration, current implementation checks for all DM calls. but it seems it is overcomplicated